### PR TITLE
Add Galaxy A50 overlay

### PR DESCRIPTION
**Pull request by milankragujevic**

### CODE REVIEW

File: Samsung/A50/res/values/notch.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<dimen name="rounded_corner_content_padding">64px</dimen>
+	<dimen name="rounded_corner_radius">92px</dimen>
```
Comment (TadiT7) [inline Samsung/A50/res/values/notch.xml]: This device has a real notch, don't just specify the corners :(

Comment (milankragujevic) [reply Samsung/A50/res/values/notch.xml]: Meh, worked well enough. I'll take a look tomorrow on other devices, then calculate it and update the PR.

**Comment by milankragujevic** [comment]
Let me redo this. Ill submit new PR when ready.

**Comment by milankragujevic** [comment]
eh

**Comment by milankragujevic** [comment]
Hi, I'm gonna redo this PR, but someone please accept it, people are emailing me constantly to send them the overlay APK for A50. I don't even have the phone anymore.

**Pull request closed without merging**